src/components/ec-option-card/ec-option-card.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <component
```

Comment:
![image](https://user-images.githubusercontent.com/5939240/220634627-dd49b398-37f3-462b-9e76-5eb25bea8b72.png)
Whad do you think? This way we could type properly our components are used in template. And code should become more structured.

Reply:
You mean this? https://vuejs.org/guide/components/provide-inject.html#provide

Reply:
Yep